atlas/AtlasMl/atlasml/app.py

```
@@ -1,31 +1,34 @@
-from fastapi import FastAPI
-import os 
 import logging
+import os
+from contextlib import asynccontextmanager
+
+from apscheduler.schedulers.background import BackgroundScheduler
+from apscheduler.triggers.interval import IntervalTrigger
+from fastapi import FastAPI
 
 from atlasml.clients.weaviate import weaviate_client
```

Pylint (3.3.7)

[error] No name 'weaviate_client' in module 'atlasml.clients.weaviate'

(E0611)